core-dev/packages/rocq-stdlib/rocq-stdlib.dev/opam

```diff
@@ -30,18 +30,13 @@ depopts: ["coq-native"]
 dev-repo: "git+https://github.com/coq/stdlib.git"
 build: [
   ["dune" "subst"] {dev}
-  [
-    "dev/with-rocq-wrap.sh"
-    "dune"
-    "build"
-    "-p"
-    name
-    "-j"
-    jobs
-    "@install"
-    "@runtest" {with-test}
-    "@doc" {with-doc}
-  ]
+  [ "find" "theories" "-name" "_*" "-prune" "-exec" "rm" "-rf" "{}" "+" ]
+  [ "dune" "build" "--root" "." "theories/All.v" ]
```

Comment:
```suggestion
  [ "dev/with-rocq-wrap.sh" "dune" "build" "--root" "." "theories/All.v" ]
```

Reply:
Building All.v doesn't care about the presence or absence of coqc AFAIK

Reply:
Indeed, don't know why I saw CI complaining about coqc not found

Reply:
I guess it does need it even if for bad reasons